### PR TITLE
Balder/deinline for gcc 8

### PR DESCRIPTION
**Pull request by baldersheim**
@toregge PR
No semantic changes intended - just cleanup before a semantic change.

### CODE REVIEW

**Comment by baldersheim** [comment]
@havardpe PR

File: searchlib/src/vespa/searchlib/common/resultset.h [hunk 1]
```diff
@@ -41,7 +41,7 @@ class ResultSet
     void mergeWithBitOverflow(HitRank default_value = default_rank_value);
 
     /* isEmpty() is allowed to return false even if bitmap has no hits */
-    bool isEmpty() const { return (_bitOverflow == NULL && _elemsUsedInRankedHitsArray == 0); }
+    bool isEmpty() const { return (_bitOverflow && (_elemsUsedInRankedHitsArray == 0)); }
```
Comment (toregge) [inline searchlib/src/vespa/searchlib/common/resultset.h]: Missing ! ?

Comment (havardpe) [reply searchlib/src/vespa/searchlib/common/resultset.h]: yes, bit overflow check was inverted.

**Review by havardpe — CHANGES_REQUESTED** [review]
logic changed (see comment). Otherwise it looks ok

File: searchlib/src/vespa/searchlib/common/resultset.h (hunk 1 above)
Comment (baldersheim) [reply searchlib/src/vespa/searchlib/common/resultset.h]: Good thing the code was never used .....

**Comment by baldersheim** [comment]
@havardpe Now the buggy code is gone together with some other unused code.

**Review by toregge — APPROVED** [review]
LGTM

**Pull request merged**